code/modules/admin/spawn_menu.dm

```diff
@@ -0,0 +1,124 @@
+GLOBAL_LIST_INIT(spawn_menus_by_ckey, list())
```

Comment:
why is this a global and not a nullable datum on the holder like our other admin tools?